networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/gossip/LibP2PGossipNetwork.java

```diff
@@ -50,4 +60,24 @@ public TopicChannel subscribe(final String topic, final TopicHandler topicHandle
     PubsubSubscription subscription = gossip.subscribe(gossipHandler, libP2PTopic);
     return new LibP2PTopicChannel(gossipHandler, subscription);
   }
+
+  @Override
+  public Map<String, Collection<NodeId>> getSubscribersByTopic() {
+    return gossip
+        .getRouter()
```

Comment:
May be consider including this feature to public Pubsub API?

Reply:
Yeah seems like a good idea.  Will merge it here now, raise a PR to jvm-libp2p and will then use the libp2p version once we pull in its next release.